### ISSUE TITLE
Target profile page rewrite: Create "Chemical Probes" detail view

### ISSUE THREAD

**Issue by andrewhercules** [opentargets/platform#334]

**Comment by andrewhercules** [comment]
### Use case(s)

1. As a user, I want to know about the chemical probes that have been developed for a given gene
2. As a user, I want to see a link to Probe Miner and explore other probes, beyond those curated for inclusion into the Platform by the Open Targets team

### Detail view

![chemical-probes](https://user-images.githubusercontent.com/7490258/49211686-09474d80-f3b8-11e8-809b-f70833bb5f06.png)

**[Full size version](https://drive.google.com/open?id=1emcDzf7NGyoMY93akPxHMp3umBfGuxA-)**
 
### Key design and interaction notes

- Please make the probe name and logo a link to the relevant external link about that probe (e.g. for BRAF, the probe GDC-0879 should link to http://www.chemicalprobes.org/GDC-0879)
- For the Probe Miner logo, please link it to the respective gene profile page on Probe Miner using the UniProt ID (e.g. for ESR1, the logo should link to https://probeminer.icr.ac.uk/#/P03372)
- Not all probes will have a note - if there is no note, please use `n/a`

### Design assets

1. Logos for various data providers are available on the [OT Core Platform Team Google Drive](https://drive.google.com/drive/folders/15cBMymLQKty6uX5jSeueLoFNza_l1HdW)
2. Colour codes are available on the [OT Core Platform Team Google Drive](https://drive.google.com/open?id=1d0JDmCznInsANZoQC77gadTf9jO0UDOo)

**Comment by peatroot** [comment]
Proposed schema:
```
type RowChemicalProbes {
  chemicalProbe: String!
  note: String
  sources: [Source!]!
}
type TargetDetailChemicalProbes {
  rows: [RowChemicalProbes!]!
  probeMinerUrl: String
}
```

**Comment by peatroot** [comment]
Now looks like the above (no icons though).

**Comment by andrewhercules** [comment]
I'm re-opening this as the implementation does not follow the spec designs. There should be two columns - one for a data table of probes (if available) and the other for a link out to Probe Miner.

Also, was there an issue in using the icons?

**Comment by peatroot** [comment]
@andrewhercules , the icons should be discussed further as we do not use icons elsewhere in the platform for other services, such as UniProt and Ensembl, so we should either do it nowhere or everywhere. Also, if we do apply the icons everywhere, there will be a lot of different colours/styles on the page. Personally, I find a simple label of the source easier to understand.

I know you designed it with a two column layout, but it means there's a lot of empty space underneath the ProbeMiner link. Happy to discuss tomorrow.

**Comment by andrewhercules** [comment]
Happy to drop the icons/logos and just use the text - we can then re-examine this once we've got a sense of some of the data where we might want to clearly indicate its source with a logo/icon (e.g. CRISPR data).

And thank you for splitting the sources - it makes it easier to link out in cases where the same probe appears in different databases (e.g. [BRD4](https://platform-app.netlify.com/target/ENSG00000141867/chemical-probes))

Once we've got the overall layout of the data table finalised (see #377), I'd like to make the note about further probes potentially available in Probe Miner more visible. 

![screen shot 2018-12-14 at 7 16 02 pm](https://user-images.githubusercontent.com/7490258/50022658-d5357480-ffd4-11e8-81c8-1ca48e84e3ae.png)
